datajs/test/browser-utils/utils.js

```diff
@@ -0,0 +1,69 @@
+// import path from 'path'
```

Comment:
Why is this all commented out?

Reply:
Not needed. These tests already exist in File Interface